app/Renderers/CalendarRenderer.php

```diff
@@ -856,6 +868,24 @@ protected function parseReminders(Collection $reminders): void
                 $this->addMultidayEvent($event, $date);
             }
         }
+
+        $start = [];
+        $end = [];
+        foreach ($this->events as $date => $events) {
```

Comment:
It doesn't work for recurring events :) 
![image](https://user-images.githubusercontent.com/3263440/207698351-1a9c7058-127c-4298-94b4-e6a2c7cf8f38.png)

I think this whole code should be in the `addMultliEvent()` function to properly loop on each occurance of an event, not just the first one (an event can appear multiple times in the same month if it's set to happen on a new moon)